TRANSLATING.md

```diff
@@ -171,10 +173,67 @@ i18nReady: true
 // Rest of the file's content is here...
 ```
 
+### Components
+
+Astro allows us to include custom components in our Markdown pages, so we take full advantage of this to improve our documentation's overall quality. Take this fragment of the `islands.md` page, which renders a diagram, as an example:
+
+```jsx
+<IslandsDiagram>
+    <Fragment slot="headerApp">Header (interactive island)</Fragment>
+    <Fragment slot="sidebarApp">Sidebar (static HTML)</Fragment>
+    <Fragment slot="main">
+        Static content like text, images, etc.
+    </Fragment>
+    <Fragment slot="carouselApp">Image carousel (interactive island)</Fragment>
+    <Fragment slot="footer">Footer (static HTML)</Fragment>
+    <Fragment slot="source">Source: [Islands Architecture: Jason Miller](https://jasonformat.com/islands-architecture/)</Fragment>
+</IslandsDiagram>
+
+```
+
+As a rule of thumb, component names, imports, and slot names (like `slot="headerApp"`) **must not be translated nor modified in any way**. Although, you're completely free to translate slotted content as you see fit.
+
+Here is the above example correctly translated:
+
+```jsx
+<IslandsDiagram>
+    <Fragment slot="headerApp">Cabeçalho (ilha interativa)</Fragment>
+    <Fragment slot="sidebarApp">Barra lateral (HTML estático)</Fragment>
+    <Fragment slot="main">
+        Conteúdo estático como texto, imagens, etc.
+    </Fragment>
+    <Fragment slot="carouselApp">Carrossel de imagens (ilha interativa)</Fragment>
+    <Fragment slot="footer">Rodapé (HTML estático)</Fragment>
+    <Fragment slot="source">Fonte: [Arquitetura em Ilhas: Jason Miller](https://jasonformat.com/islands-architecture/)</Fragment>
+</IslandsDiagram>
+```
+
+Note that some of our components' labels are instead translated inside the language's respective `i18n/` files, as we explain in the [Translation Structure section](#2-ui-text).
+
+### Asides
+
+Most of our pages include stylish tip/note/caution blocks called "asides". As they're repeatedly used throughout the docs, we use a custom syntax to author them (and later replace with components during build), as you can see in the example below:
+
+```
+:::tip[Online previews]
+Prefer to try Astro in your browser? Visit [astro.new](https://astro.new/) to browse our starter templates and spin up a new Astro project without ever leaving your browser.
+:::
+```
+
+Some asides contain custom inline labels inside `[square brackets]`, like the example above. You're free to translate these, as well as the text inside the backticks. Although, you **must not translate** the aside's type (like `:::tip`), as these are translated in the language's respective `i18n/nav.ts` file.
```

Comment:
Do you mean inside the `:::`-delimited block rather than backticks?

```suggestion
Some asides contain custom inline labels inside `[square brackets]`, like the example above. You're free to translate these, as well as the text inside the aside block. Although, you **must not translate** the aside's type (like `:::tip`), as these are translated in the language's respective `i18n/nav.ts` file.
```

Reply:
Exactly, looks like my mind was thinking about code blocks instead of aside blocks 😅